### ISSUE TITLE
error: err, you are not an admin!

### ISSUE THREAD

**Issue by LAVACODE** [roshan07/AppointmentBooking-MEAN-Stack-App#1]
Hello after starting this app I got errors in console.

err, you are not an admin!
auth.service.ts:150 err, you are not an admin!
auth.service.ts:150 err, you are not an admin!
auth.service.ts:150 err, you are not an admin!
auth.service.ts:150 err, you are not an admin!
auth.service.ts:150 err, you are not an admin!
auth.service.ts:150 err, you are not an admin!
auth.service.ts:150 err, you are not an admin!

can't register user or do anything

**Comment by roshan07** [comment]
hey, you need to setup MongoDB database locally in your machine with the database name and then provide your own secret key for that.

**Comment by LAVACODE** [comment]
I know and I did it.
Should I add manually to DB user with admin role?

**Comment by roshan07** [comment]
yes, you need to add manually a user with admin role in MongoDB.

**Comment by LAVACODE** [comment]
it works! :)

**Comment by roshan07** [comment]
Great, Hoo Yaah!!! Happy Coding.
